packages/packages.csv

```diff
@@ -4,7 +4,6 @@ Microsoft.VisualStudio.Web.CodeGeneration.Tools,ship
 Microsoft.Extensions.SecretManager.Tools,ship
 Microsoft.DotNet.Watcher.Tools,ship
 Microsoft.DotNet.Watcher.Core,ship
-Libuv,ship-ext
```

Comment:
Since we no longer build the package Coherence doesn't see it. We'll need to figure out what we need with this package for RTM (maybe deal with it manually?)